### PR TITLE
Website main page changes and added past events

### PR DESCRIPTION
**Pull request by wjiang42**
Added new photos for the main page, reduced background tint and added AB Underground mxmtoon and Phoebe Bridgers. 

I am aware that many old files remain, and that photos in /services are unchanged.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -68,7 +68,7 @@ build-iPhoneSimulator/
 
 # for a library or gem, you might want to ignore these files since the code is
 # intended to run in multiple environments; otherwise, check them in:
-# Gemfile.lock
+Gemfile.lock
```
Comment (DaAwesomeP) [inline .gitignore:71]: I don't think this is right... why this change?

File: assets/img/carousel/astrasse.JPG
Comment (DaAwesomeP) [inline assets/img/carousel/astrasse.JPG:1]: Maybe rename this to which event it is from so that the this person's name is not baked into the website and it is easier to figure out where the photo came from later

File: assets/img/carousel/haichuas_f23.JPG
Comment (DaAwesomeP) [inline assets/img/carousel/haichuas_f23.JPG:1]: Same

File: assets/img/carousel/gskam_f23.JPG
Comment (DaAwesomeP) [inline assets/img/carousel/gskam_f23.JPG:1]: Same

File: assets/img/carousel/esprite.JPG
Comment (DaAwesomeP) [inline assets/img/carousel/esprite.JPG:1]: Just my opinion--this image is cool if you know what you are looking at but otherwise just looks like a light.

**Review by DaAwesomeP — CHANGES_REQUESTED** [review]
Only one thing that is particularly blocking (`.gitignore`).

Just a small editorial note--for the images, think about who the homepage target audience is. All of these images are showing are from event and focuses on the Techies. This is great for targeting new members but maybe less so people submitting event requests for a variety of event types (dance, talent show, fashion show, corporate style, etc.) where they care more about the end result and seeing AB Tech's capabilities and "showreel" (i.e. "look at all the amazing high-production value events we do!"). Feel free to ignore me, but my recommendation would be to mix techies + cool event photos of a variety of event types.

File: .gitignore (hunk 1 above)
Comment (wjiang42) [reply .gitignore:71]: On my local machine the Gemfile keeps adding arm support and it was getting annoying, but I should change it back. Out of curiosity, is it because I have an M1 Mac?

File: assets/img/carousel/esprite.JPG
Comment (wjiang42) [reply assets/img/carousel/esprite.JPG:1]: Oof that is such a good point

**Comment by wjiang42** [comment]
When I was looking through drive/Photos for photos, I was mainly looking for photos taken from a proper camera (i.e. not phone camera), but I will look for some photos of more events. Thanks for the feedback!

File: .gitignore (hunk 1 above)
Comment (DaAwesomeP) [reply .gitignore:71]: Yeah probably M1 related. I can look into why that happens, though you shouldn't have to run `bundle install` every time you build the website. The `Gemfile.lock` file ensures that the same exact dependency versions get installed on each machine (for reproducibility of builds).

**Comment by DaAwesomeP** [comment]
> When I was looking through drive/Photos for photos, I was mainly looking for photos taken from a proper camera (i.e. not phone camera), but I will look for some photos of more events. Thanks for the feedback!

TBH, the latest generations of iPhones take very good photos that I don't think anyone will be able to tell the difference. Bridget takes photos with a nicer camera (and I think she has visited a few times recently so there should be some more recent events with her photos).

File: .gitignore (hunk 1 above)
Comment (wjiang42) [reply .gitignore:71]: Gotcha, for now I just git restore the Gemfile, which isn't the worst thing in the world, so not pressing, but would love to know if there's a way to stop it from doing that without having to change the gitignore

**Comment by wjiang42** [comment]
Personally I feel like modern phone cameras still struggle with low light environments, which is a lot of our events, so for this round of photos I only used pictures from Heidi and Bridget's cameras, but I'll start look at phone photos too.

**Review by DaAwesomeP — CHANGES_REQUESTED** [review]
Hi sorry for the delay; this one slipped through my inbox. Feel free to ping me next time.

One typo: the file `concert_f23_mons.JPG` has the file extension capitalized. This might actually work on Windows and macOS because I think those filesystems might not be case-sensitive, but the server uses a case-sensitive file system (as does my computer, so one image in this carousel fails for me).

One small request: In the future please split up unrelated changes into different pulls (i.e. for this pull the HoTs page and services page)

**Comment by DaAwesomeP** [comment]
The repsonsive sizing on the HoTs page is not working properly now.

Before:
image

After:
image

**Comment by DaAwesomeP** [comment]
The `px-4` added to the layout template is also causing some smushing on mobile; you might want to make this responsive.

**Comment by wjiang42** [comment]
Oof apologies for putting so many changes here, got lazy haha

File: _layouts/default.html [hunk 2]
```diff
@@ -22,7 +22,7 @@
     {% include nav.html %}
   
     <main class="{% if page contains 'main_classes_override' %}{{ page.main_classes_override }}{% else %}flex-shrink-0 flex-grow-1 w-100{% endif %}" role="main">
-      <div class="container content-block d-block mx-auto pb-3">
+      <div class="container content-block d-block mx-auto pb-3 pxmd-4">
```
Comment (DaAwesomeP) [inline _layouts/default.html:25]: This should be `px-md-4` else it has no effect.

Comment (wjiang42) [reply _layouts/default.html:25]: Damn thought i fixed that
